pkg/server/receive.go

```diff
@@ -56,5 +56,6 @@ func (srv *Server) handleReceive(w http.ResponseWriter, r *http.Request) {
 		srv.sendError(ctx, w, &HTTPError{StatusCode: http.StatusInternalServerError, Err: err})
 	}
 	srv.UpdateGlobalMetrics(ctx)
-	w.WriteHeader(202)
+	w.WriteHeader(http.StatusAccepted)
+	_, _ = fmt.Fprint(w, "Webmention accepted")
```

Comment:
You don't even need to declare the return values if you're not planning to use any of them :)

Reply:
IntelliJ IDEA suggests setting them :sweat_smile:

Reply:
It makes sense if you explicitly want to indicate that you knowing possible error conditions but explicitly opted to ignore them. 👍🙂